pallets/anonymity-mining-claims/src/lib.rs

```diff
@@ -135,31 +158,40 @@ pub mod pallet {
 		type ForceOrigin: EnsureOrigin<Self::RuntimeOrigin>;
 	}
 
+	/// The map of trees to the maximum number of anchor edges they can have
+	#[pallet::storage]
+	#[pallet::getter(fn max_edges)]
+	pub type MaxEdges<T: Config<I>, I: 'static = ()> = StorageValue<_, u32, ValueQuery>;
```

Comment:
This currently doesn't match the comment. Is this meant to be a map? Also since it's rewards pallet we should probably map `ResourceID -> u32` to indicate the size of the bridge that a resource ID in question has.